modules/swagger-codegen/src/main/resources/swift/Models.mustache

```diff
@@ -11,19 +11,8 @@ protocol JSONEncodable {
 }
 
 public enum ErrorResponse : ErrorType {
-    case RawError(Int, NSData?, ErrorType)
-    {{#operations}}
-        xxx
-    {{#operation}}
-        yyy
-        {{{operationId}}}
-    {{#responses}}{{^isDefault}}
-        zzz
-    case {{{operationId}}}{{{code}}}(Error)
-    {{/isDefault}}
-    {{/responses}}
-    {{/operation}}
-    {{/operations}}
+    case RawError(Int, NSData?, ErrorType){{#apiInfo.apis}}{{#operations}}{{#operation}}{{#responses}}{{^isDefault}}
+    case {{{operationId}}}{{{code}}}({{{dataType}}}){{/isDefault}}{{/responses}}{{/operation}}{{/operations}}{{/apiInfo.apis}}
```

Comment:
Sorry, I did not mean to add this second commit to the PR. Let me remove it. We can debate that later commit once this simpler one is merged.